setup.py

```diff
@@ -16,27 +16,24 @@
 
 setup(
     name="pulser",
-    version="0.0.1",
+    version="0.1.0",
```

Comment:
I'm seeing `0.1.1` on TestPypi , is this relevant? I.e. does this version numbering correspond to the real Pypi numbering?

Reply:
This is the right one, I had to bump it up in TestPypi because I had to upload more than once and it doesn't let you repeat a version.